test/unittests/instantiate_test.cpp

```diff
@@ -382,6 +382,27 @@ TEST(instantiate, imported_globals_mismatched_mutability)
         "global 0 mutability doesn't match module's global mutability");
 }
 
+TEST(instantiate, DISABLED_imported_globals_mismatched_type)
```

Comment:
To ebable this, we'll need to change `ExternalGlobal` type to include value type.

Also, it looks to be not covered in spec tests.

Reply:
Did you add this to the list?

Reply:
Added.